docs/zh-cn/tutorials/machine-learning-overview.md

```diff
@@ -79,7 +79,7 @@ predictBookPrice('Dive into Pipcook'); // 199.0
 ```js
 {
   "plugins": {
-    "package": "@pipcook/plugins-mnist-data-collect",
+    "dataCollect": "@pipcook/plugins-mnist-data-collect",
```

Comment:
package field should be inside dataCollect

Reply:
Done.